spaces/S000086/properties/P000200.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000086
+property: P000200
+value: false
+refs:
+  - zb: "1044.55001"
+    name: Algebraic Topology (Hatcher)
+---
+
+The map $(x,i)\mapsto(x,0)$ for $x\in\mathbb R\setminus\{0\}$, $i\in\{0,1\}$ and fixing the two zeros is a retraction onto {S83}. Since {S83|P200}, it follows by Proposition 1.17 of {{zb:1044.55001}} that $X$ is not simply connected.
```

Comment:
```suggestion
The map $(x,i)\mapsto(x,0)$ for $x\in\mathbb R\setminus\{0\}$, $i\in\{0,1\}$ and fixing the two "zero points" is a retraction onto {S83}. Since {S83|P200}, it follows by Proposition 1.17 of {{zb:1044.55001}} that $X$ is not simply connected.
```
Seems clearer to me.